extensions/answer_summarizers/models_test.py

```diff
@@ -473,7 +480,9 @@ def test_top_answers_by_categorization_cannot_classify_invalid_category(self): #
             'submitted_answer_list': dummy_submitted_answer_list
         }
 
-        with self.assertRaisesRegexp(
-            Exception, 'Cannot aggregate answer with unknown rule'):
+        actual_state_answers_calc_output = (
             calculation_instance.calculate_from_state_answers_dict(
-                state_answers_dict)
+                state_answers_dict))
+        actual_calc_output = (
+            actual_state_answers_calc_output.calculation_output)
+        self.assertEqual(actual_calc_output, {})
```

Comment:
Nothing comes to mind since the output is defined based on categories and we filter out categories which are invalid.